core/src/main/kotlin/io/specmatic/core/pattern/Row.kt

```diff
@@ -95,6 +95,9 @@ data class Row(
         if(requestBodyJSONExample == null)
             return this
 
+        if(requestBodyJSONExample.jsonObject is JSONArrayValue)
```

Comment:
@joelrosario , require your help here. For now we have introduced this condition to make a test pass. We are not much sure if this is right. We'll discuss.